### DISCUSSION TITLE
deforum

### DISCUSSION THREAD

**Discussion by fedepalu2** [rupeshs/fastsdcpu#113] [Ideas]
Would be possible add deforum extension?

Thanks

**Comment by rupeshs** [comment]
Not planned , @fedepalu2  see this https://github.com/deforum-art/deforum-stable-diffusion/issues/189

**Comment by fedepalu2** [comment]
Dear
I see.

Thanks
Federico

Il dom 24 dic 2023, 04:59 Rupesh Sreeraman ***@***.***> ha
scritto:

**Comment by fedepalu2** [comment]
Hi dear!

Happy Christmast!

What about "Stable Video Diffusion" in fastcpu?

Do you  think it is possible to use Openvino with "Stable Video Diffusion"?

Have a nice day
Federico

Il giorno dom 24 dic 2023 alle ore 10:34 Federico Palumbo <
***@***.***> ha scritto:

**Reply by rupeshs** [reply]
Happy Christmas, OpenVINO does not support text-to-video
